src/TestUtils/src/Microsoft.Maui.IntegrationTests/BaseBuildTest.cs

```diff
@@ -6,6 +6,9 @@ public class BaseBuildTest
 		public const string DotNetCurrent = "net7.0";
 		public const string DotNetPrevious = "net6.0";
 
+		public const string MauiVersionCurrent = "7.0.86"; // this should not be the same as the last release
```

Comment:
So what should this be ? this case we have 7.0.92 , is this previous the current stable?

Reply:
I think this could be *any* previous stable release. Is it on the `dotnet7` feed?

Reply:
This will never work on main... main is a net8 style with net7 tfm. Literally not a valid scenario. Let me rebase on net8.